packages/react/src/index.ts

```diff
@@ -201,11 +201,23 @@ export {
   ContentComponentProps,
   SizeValue,
 } from './lib'
-export { ShorthandRenderer } from './types'
+export { ShorthandValue, ShorthandRenderer } from './types'
 
 //
 // FocusZone
 //
-import { getFirstTabbable, getLastTabbable } from './lib/accessibility/FocusZone/focusUtilities'
-export const FocusZoneUtilities = { getFirstTabbable, getLastTabbable }
+import {
+  getFirstTabbable,
+  getLastTabbable,
+  getNextElement,
+  getPreviousElement,
+  focusAsync,
+} from './lib/accessibility/FocusZone/focusUtilities'
+export const focusZoneUtilities = {
```

Comment:
just one thing - shouldn't it be a breaking change, given this code on the client's side:

```jsx
import { FocusZoneUtilities, ... } from '@stardust-ui/react'
```

Reply:
Oh, I see, great catch 👍